src/daemon/pulse/pulse-network.c

```diff
@@ -499,7 +499,7 @@ void pulse_network_do(bool extended __maybe_unused) {
                     "milliseconds",
                     "netdata",
                     "pulse",
-                    PULSE_NETWORK_CHART_PRIORITY + 1,
+                    PULSE_NETWORK_CHART_PRIORITY + 4,
```

Comment:
A minor suggestion which is not necessary to address in this PR. We currently define priorities in two locations:

- This file (e.g., `#define PULSE_NETWORK_CHART_PRIORITY 130150`)
- `src/collectors/all.h`

For better consistency, I recommend consolidating these priority definitions in `src/collectors/all.h`, which already serves as the central location for priority values.

Reply:
can be done in a future PR